forui/test/src/test_scaffold.dart

```diff
@@ -4,6 +4,26 @@ import 'package:flutter/material.dart';
 import 'package:forui/forui.dart';
 
 class TestScaffold extends StatelessWidget {
+  static final blueScreen = FThemeData.inherit(
+    colorScheme: const FColorScheme(
+      brightness: Brightness.light,
+      disabledColorLightness: 0.1,
+      background: Colors.lightBlue,
```

Comment:
I think it could be neater to define a color for the blue tests rather then using Material's one. Could be problematic if they decide to make any changes (eg. shade of blue/depreciate/etc).